docs/user-guides/devcontainers/index.md

```diff
@@ -38,62 +25,64 @@ which allows for extensive customization of your development setup.
 When a workspace with the dev containers integration starts:
 
 1. The workspace initializes the Docker environment.
-1. The integration detects repositories with a `.devcontainer` directory or a
-   `devcontainer.json` file.
-1. The integration builds and starts the dev container based on the
-   configuration.
+1. The integration detects repositories with a `.devcontainer` directory or a `devcontainer.json` file.
+1. The integration builds (or rebuilds) and starts the dev container based on the configuration.
 1. Your workspace automatically detects the running dev container.
+1. If the configuration changes, the workspace prompts you to rebuild the dev container.
 
 ## Features
 
-### Available Now
+### Detection & Lifecycle
 
-- Automatic dev container detection from repositories
-- Seamless dev container startup during workspace initialization
-- Integrated IDE experience in dev containers with VS Code
-- Direct service access in dev containers
-- Limited SSH access to dev containers
+- Automatic discovery of `.devcontainer` configs.
+- Change detection with rebuild prompts.
+- Rebuild containers with one click from the Coder dashboard or from the CLI.
 
-### Coming Soon
+### Connectivity
 
-- Dev container change detection
-- On-demand dev container recreation
-- Support for automatic port forwarding inside the container
-- Full native SSH support to dev containers
+- Full SSH access directly into dev containers (`coder ssh --container ...`).
+- Automatic port forwarding based on `appPort`, `forwardPorts`, or `docker-compose.yml`.
 
-## Limitations during Early Access
+## Personal overrides
 
-During the early access phase, the dev containers integration has the following
-limitations:
+To add tools or tweaks that enhance your personal experience, create a `devcontainer.local.json` file in the same
+directory as the project’s `devcontainer.json`:
```

Comment:
I have no idea if this works. Is it a feature of `@devcontainers/cli`?

Reply:
Blink seems terribly convinced that it's dealt with in https://github.com/devcontainers/cli/blob/main/src/spec-node/imageMetadata.ts#L99 but I haven't been able to test it or even follow its logic

it is a question I still have though: "can it handle additional local configuration/how" but I can leave this for later